build-system/tasks/performance/rewrite-analytics-tags.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+const {
+  CONTROL,
+  EXPERIMENT,
+  getLocalVendorConfig,
+  urlToCachePath,
+} = require('./helpers');
+const {JSDOM} = require('jsdom');
+
+/**
+ * Return local vendor config.
+ *
+ * @param {string} vendor
+ * @return {Object}
+ */
+async function getVendorConfig(vendor) {
+  return JSON.parse(await getLocalVendorConfig(vendor));
+}
+
+/**
+ * Fetch the vendor config and merge it with inline config.
+ * Work around for fetching real vendor config that gets blocked
+ * by AMP-analytics. Order of merging matters. Uses local config
+ * for both CONTROL and EXPERIMENT.
+ *
+ * @param {Element} tag
+ * @param {Object} script
+ * @return {Object}
+ */
+async function maybeMergeAndRemoveVendorConfig(tag, script) {
+  if (tag.hasAttribute('type')) {
+    const vendor = tag.getAttribute('type');
+    tag.removeAttribute('type');
+    const vendorConfig = await getVendorConfig(vendor);
+    // Shallow copy is ok
+    const x = Object.assign(vendorConfig, script);
```

Comment:
Let's keep this for now, but you'll need a deep merge. Is it possible to import method from amp-analytics/config.js?

Reply:
I think generally we don't want to rely on v0 for stuff within our build-system? @estherkim can you confirm?

Reply:
It's okay to import the method. We import from extensions when compiling parsers, for one. I also found some instances that import `src/mode`, `src/layout` and more.

Reply:
Ok, thanks @estherkim. I'll just add it in now.

Reply:
I don't see the change. Could you add a TODO?